owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.kt

```diff
@@ -1026,6 +1051,62 @@ class FileDisplayActivity : FileActivity(),
         }
     }
 
+    private fun showConflictDecisionDialog(
+        uiResult: UIResult.Success<List<OCFile>>,
+        data: List<OCFile>,
+        replace: MutableList<Boolean?>,
+        launchAction: (List<OCFile>, List<Boolean?>) -> Unit,
+    ) {
+        val context = this@FileDisplayActivity
+        if (!uiResult.data.isNullOrEmpty()) {
+            data.forEachIndexed { pos, file ->
+                AlertDialog.Builder(context)
+                    .setTitle(R.string.file_already_exists)
+                    .setMessage(String.format(context.getString(R.string.conflict_description), file.fileName))
+                    .setPositiveButton(R.string.welcome_feature_skip_button) { _, _ ->
```

Comment:
Putting the same order from the web

Reply:
Ok, although the left button continues being separated from the other 2 now it makes more sense since "Skip" has no action (neutral button, as its name suggests) and "Replace" and "Keep both" do have actions 👍. In any case, QA will have the last word here @jesmrec 😁